charts/firecloudui/templates/config/_prometheusJmx.yaml.tpl

```diff
@@ -0,0 +1,5 @@
+{{- /* Configuration for Prometheus JMX exporter javaagent */ -}}
+{{- define "firecloudui.config.prometheusJmx" -}}
+rules:
+- pattern: ".*"
+{{ end -}}
```

Comment:
since firecloud ui isn't a jvm app you don't need this file at all.

Reply:
FINE. IT IS DEAD TO ME. Happy?

Reply:
Removed file.